src/main/java/net/openhft/chronicle/queue/impl/single/SCQIndexing.java

```diff
@@ -658,12 +658,12 @@ public boolean indexable(long index) {
         return (index & (indexSpacing - 1)) == 0;
     }
 
-    public long lastSequenceNumber(@NotNull ExcerptContext ec)
+    public long lastSequenceNumber(@NotNull ExcerptContext ec, boolean approximate)
```

Comment:
We should provide a compatible `stSequenceNumber(@NotNull ExcerptContext ec)` so that old code using the signature of the old method can compile. This method can then call the new method with an appropriate value of the `approximate` parameter.

Reply:
It is a non-public class, I don't think that we need to care about code calling it, unless it's our code.

Reply:
It's package private and the method is not part of any interface.

Reply:
Great. That's perfect. We do not have to do anything.